cmd/cluster/server.go

```diff
@@ -21,44 +24,56 @@ type Cluster struct {
 
 // newCluster will create a new Cluster with default values.
 func newCluster() *Cluster {
+	// 1.1.1.1 is used for route lookup, to determine the local interface that would
+	// be used to reach this address.  It doesn't actually talk to that address.
 	local, err := nodes.LocalAddress("1.1.1.1:1")
```

Comment:
Does it need to be an outbound addr in order to get the interface addr that is used to connect to the outside world?

ie. does trying to dial `127.0.0.1` give you an interface that isn't compatible?

Reply:
Yeah, it would return 127.0.0.1:
```
1.1.1.1:1111 -> 10.0.2.15 (default)
127.0.0.1:1234 -> 127.0.0.1
192.168.56.1:1234 -> 192.168.56.101 (virtualbox internal network)
```